remappings.txt

```diff
@@ -0,0 +1,6 @@
+@openzeppelin/contracts/=lib/openzeppelin-contracts-upgradeable/contracts/
```

Comment:
Why was this file not needed previously?

Reply:
This is used by the hardhat preprocessor to change the import to their correct path before compiling them. This so that we can still use `forge build` without changing any imports and for hardhat to build the project without changing anything